src/array-to-integer.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('lodash');
+
+const fromIntegerArray = function(integerArray) {
+  return getValue(_.uniq(integerArray), 'number', (function(_this) {
+    return function(bitmap, value, index) {
+      if (value > 0) {
+        return bitmap.value += Math.pow(2, (Math.floor(value)) - 1);
```

Comment:
You can remove unnecessary return
```suggestion
        bitmap.value += Math.pow(2, (Math.floor(value)) - 1);
```